blog/models.py

```diff
@@ -0,0 +1,53 @@
+from django.conf import settings
+from django.contrib.auth.models import AbstractUser
+from django.db import models
+from django.db.models import CharField
+
+
+class User(AbstractUser):
+    def __str__(self) -> str:
+        return f"{self.first_name} {self.last_name}"
+
+
+class Post(models.Model):
+    objects = None

```

Comment:
The `objects` attribute is set to `None`, which is unnecessary. It should be removed to allow Django to use the default manager.